### PR TITLE
[Data Science] Data preparation

### LINKED ISSUE #61: Data preparation
Merge data with the help of a practical example and drop duplicates.

### PR DESCRIPTION
**Pull request by JakobKlotz**
Closes #61 

# Topics

- drop duplicates
- merge methods

### CODE REVIEW

File: docs/data-science/data/preparation.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+  left `DataFrame` (data 1) and the matched rows from the right `DataFrame` 
+  (data 2).
+- **Right join**: The resulting `DataFrame` will contain all rows from the 
+  right `DataFrame` (data 2) and the matched rows from the left `DataFrame` 
+  (data 1).
+- **Inner join**: The resulting `DataFrame` will contain only the rows that 
+  have matching values in both `DataFrame`s.
+- **Outer join**: The resulting `DataFrame` will contain all rows from both 
+  `DataFrame`s.
+
```
Comment (manuf3006) [inline docs/data-science/data/preparation.md:257]: I like how you described the different merges. Based on our audience: maybe we can give a small minimal example for each? Or link a website, where a minimal example is shown?

File: docs/data-science/data/preparation.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+```title=">>> Output"
+Shape of data_merged: (3928, 21)
+```
+
+We end up with `#!python 3928` customers that are present in both data sets.
+Lastly, we can write the merged data set to a new file. Let's use a common 
+format :fontawesome-solid-arrow-right: `csv` with the default `,` as separator.
+
+```python
+data_merged.to_csv("data/bank-merged.csv", index=False)
```
Comment (manuf3006) [inline docs/data-science/data/preparation.md:283]: Maybe we should explain, what index=False mean (or give it as a task)

File: mkdocs.yml [hunk 3]
```diff
@@ -139,6 +139,13 @@ nav:
       - Regression Analysis:
         - statistics/regression/LinearRegression.md
 
+  - Data Science:
+      - data-science/index.md
+
+      - Data Preparation & Preprocessing:
+        - data-science/data/preparation.md
+        - data-science/data/preprocessors.md  # TODO: different transformations
```
Comment (manuf3006) [inline mkdocs.yml:147]: The preprocessor.md file is not contained in this pull request. I don't know, if this was on purpose or by accident? To due this fact, in the left menu 'None' will be shown

File: docs/data-science/data/preparation.md (hunk 1 above)
Comment (JakobKlotz) [reply docs/data-science/data/preparation.md:257]: I've extended the section on merge methods, to explain provide some additional examples and an 'in-depth' look.

File: mkdocs.yml (hunk 3 above)
Comment (JakobKlotz) [reply mkdocs.yml]: My bad, I've removed the file from the navigation. It will be part of the next PR. 😄

**Pull request merged**